src/pages/Admin/Details/Form/DetailsForm.styles.ts

```diff
@@ -9,6 +9,11 @@ export const FormInnerWrapper = styled.div`
   padding: 1em 5em;
 `;
 
+export const FormWrapper = styled.form`
+  height: 100vh;
+  overflow-y: scroll;
+`;
+
```

Comment:
Please don't add more styled components, we're in the process of replacing them with SCSS+BEM per component